pom.xml

```diff
@@ -167,4 +167,12 @@
         </dependencies>
     </dependencyManagement>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.jgrapht</groupId>
+            <artifactId>jgrapht-core</artifactId>
+            <version>1.5.1</version>
```

Comment:
The version is already in the pom parent (powsybl-core). So you can remove the `version` tag